### DISCUSSION TITLE
Unable to locate factory if using dump

### DISCUSSION THREAD

**Discussion by ejunker** [laravel/framework#32990] [General]
- Laravel Version: 7.13.0
- PHP Version: 7.3.4
- Database Driver & Version: no database used

### Description:

In my feature tests I like to dump the response to help with debugging. I had a test that kept getting the error `Unable to locate factory` but if I removed the dump it would work. I am not sure if this is a Laravel bug, Symfony VarDumper bug or maybe a bug with facade/ignition but thought I'd start with Laravel and see if anybody has ideas on what is going on.

### Steps To Reproduce:

I was able to reproduce the issue in a fresh Laravel 7 app. Here are steps to reproduce.

```
git clone https://github.com/laravel/laravel.git dump-test
cd dump-test
cp .env.example .env
composer install
php artisan key:generate
```

Create `tests/Feature/DumpTest.php` like this
```php
<?php

namespace Tests\Feature;

use App\User;
use Tests\TestCase;

class DumpTest extends TestCase
{
    public function test_one()
    {
        $this->assertTrue(true);
    }

    public function test_two()
    {
        $this->get('/')
            ->dump()
            ->assertSuccessful();

        factory(User::class)->make();
    }
}

```
Run `php artisan test` and it will fail.
Run `./vendor/bin/phpunit tests/Feature/DumpTest.php` and it will fail.
Run `./vendor/bin/phpunit tests/Feature/DumpTest.php --filter two` and it will pass.
Comment out the `->dump()` and run `php artisan test` and it will pass.

I also noticed if I go into `vendor/facade/ignition/src/DumpRecorder/DumpRecorder.php`  in the `register()` method and comment out the following that the test will pass.

```php
        $multiDumpHandler->addHandler(function ($var) {
            //$this->app->make(DumpHandler::class)->dump($var);
        });
```

**Comment by driesvints** [comment]
Moving to discussions.

**Reply by ejunker** [reply]
@driesvints thanks for taking a look at my issue. Discussions is relatively new. How is Laravel using Discussions? If my bug report is moved to Discussions does that mean it is unlikely to be fixed? How do you determine what stays an issue and what gets moved to Discussions?
